src/Microsoft.Health.Fhir.Core/Features/KnownQueryParameterNames.cs

```diff
@@ -90,6 +90,8 @@ public static class KnownQueryParameterNames
 
         public const string IgnoreSearchParamHash = "_ignoreSearchParamHash";
 
+        public const string FeedRange = "_feedRange";
```

Comment:
I struggled with the decision to make this a query parameter or not. In the end I did because the surrogate id parameters are similar.

If this shouldn't be a query param should the surrogate id params be here either?

Reply:
Given all the params I've been adding in this area (😂), the only other solution that comes to mind is separating these out from being passed as "query parameters" to another object that has "system request modifiers" as properties. Cosmos and SQL could have their own derived classes (no `FeedRange` for SQL for example).

Not sure if it's worth the work. Thinking of something like

*SystemRequestModifiers.cs*
``` csharp
{
    public class SystemRequestModifiers
    {
        public bool UseSmartCompartmentDefinitions {get; set; }
        public ResourceTypeVersion useSmartCompartmentDefinitions {get; set; }
        public bool IsAsyncOperation {get; set; }
        ....
    }
}
```

*SqlSystemRequestModifiers.cs*
``` csharp
{
    public class SqlSystemRequestModifiers : SystemRequestModifiers
    {
        public long GlobalStartId {get; set; }
        public long GlobalEndIt {get; set; }
        public long GlobalSurrogateStartId {get; set; }
        public long GlobalSurrogateEndId {get; set; }
    }
}
```

*CosmosSystemRequestModifiers.cs*
``` csharp
{
    public class CosmosSystemRequestModifiers : SystemRequestModifiers
    {
        public FeedRange FeedRange {get; set; }
    }
}
```